src/main/java/WayofTime/alchemicalWizardry/client/nei/NEIMeteorRecipeHandler.java

```diff
@@ -61,10 +66,44 @@ public CachedMeteorRecipe(MeteorParadigm meteor, ItemStack focusStack) {
                     row++;
                 }
 
-                if (focusStack != null && matchItem(focusStack, stack)) {
+                if (matchItem(focusStack, stack)) {
                     this.focus = new Point(xPos - 1, yPos - 1);
                 }
             }
+
+            if (fillerChance > 0) {
+                if (col != 0) {
+                    col = 0;
+                    row++;
+                }
+
+                List<MeteorParadigmComponent> sortedFiller = new ArrayList<>(meteor.fillerList);
```

Comment:
This is not actually sorted.

Reply:
Addressed by [9c0be69](https://github.com/GTNewHorizons/BloodMagic/pull/60/commits/9c0be69a1719c5f63d1fdb3c105e7476080c8533)